packages/@okta/vuepress-site/docs/guides/sign-into-web-app/main/index.md

```diff
@@ -20,9 +20,9 @@ This guide shows you how to use Okta as the user store for your server-side web
 
 **What you need**
 
-* An Okta Developer Edition organization. Don't have one? [Create one for free](https://developer.okta.com/signup).
-* Knowledge of building server-side Web applications.
-* A project or application that you want to add authentication to.
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* Knowledge of building server-side Web applications
+* A project or application to add authentication to.
```

Comment:
Should this bullet point have a period? I'm trying to interpret the nut facts guidelines :)

Reply:
Yes it should :-)